server/template.js

```diff
@@ -110,8 +136,14 @@ export default ({ url, bodyClass, title, description }) => marinate`
   <!-- Polyfills -->
   <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Object.entries,Array.from,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Map,Set,requestAnimationFrame,Array.prototype.@@iterator"></script>
 
+<<<<<<< HEAD
+  <script src="${manifest['vendor.js']}"></script>
+  <script src="${manifest['app.js']}"></script>
+</body>
+=======
   <!-- Source Code -->
   ${scriptsHTML}
```

Comment:
`scriptsHTML` handles the script tags, you just need to update the array below.

Reply:
Couldn't get you here. Which array are we talking about?

Reply:
Check at the bottom of the file and look for `scripts` array. 

https://github.com/KhalisFoundation/sttm-web/blob/dev/server/template.js#L173

Reply:
@bogas04 Can you check now, if all the required changes are done?